SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -54,9 +56,18 @@
     -->
   <Target Name="SonarQubeCategoriseProject"
           BeforeTargets="CoreCompile"
-          Condition=" $(SonarQubeTempPath) != '' AND $(SonarQubeTestProject) == '' " >
+          Condition=" $(SonarQubeTempPath) != '' " >
 
+    <!-- Fakes detection -->
     <PropertyGroup>
+      <IsFakesProject Condition="$(AssemblyName.EndsWith('.fakes', System.StringComparison.OrdinalIgnoreCase))">true</IsFakesProject>
```

Comment:
Do they work in the same way? i.e. via targets that generate projects on the fly that spawn MSBuild?